source/macos/AdaptiveCards/AdaptiveCards/Renderers/ColumnSetRenderer.swift

```diff
@@ -4,50 +4,35 @@ import AppKit
 class ColumnSetRenderer: BaseCardElementRendererProtocol {
     static let shared = ColumnSetRenderer()
     
+    struct Constants {
+        static let kFillColumnViewVerticalLayoutConstraintPriority = NSLayoutConstraint.Priority.defaultLow - 1
+    }
```

Comment:
Add a comment saying this value is always 249 maybe

Reply:
may we wont need that, we mention defaultLow which is always 250 so.